axelrod/strategies/cycler.py

```diff
@@ -77,6 +78,8 @@ def strategy(self, opponent):
 
 
 class CyclerDC(Cycler):
+
+    name = 'Cycler DC'
```

Comment:
Do we need to set the name explicitly in addition to the dynamic assignment above?